src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier randomColor = new ColorSupplier();
+
+        randomBall.setNumber((int) (Math.random() * 100));
```

Comment:
Magic numbers should be replaced with named constants. In this case, the number '100' should be extracted into a constant with a clear name indicating its purpose.